src/workerd/util/sqlite.c++

```diff
@@ -61,6 +62,41 @@ namespace workerd {
 // Version of `SQLITE_CALL` that can be called after inspecting the error code, in case some codes
 // aren't really errors.
 
+#define VALUE_GATE(variable, maybeNull, ...) do { \
```

Comment:
In C++ we generally avoid complex macros like this. Instead, C++ templates usually provide a better answer. That said, in this case I'm not sure if you really need either, I think a plain function would work, perhaps using `std::initializer_list` to take variadic arguments (or equivalently `kj::ArrayPtr`).